stories/src/annotation.stories.tsx

```diff
@@ -471,8 +471,8 @@ storiesOf('Annotations', module)
 
     const annotationsInput =
       columnKey === TIME
-        ? [currentTime.split(), endTime.split()]
-        : [currentValue.split(), endValue.split()]
+        ? [[currentTime], [endTime]]
+        : [[currentValue], [endValue]]
```

Comment:
`"foo".split()` returns `["foo"]`. This is valid JS, but TypeScript apparently has a problem with it.